packages/ui/src/routes/setup/SetupDonePage.tsx

```diff
@@ -7,13 +7,26 @@ import PageLayout from "../../components/PageLayout"
 
 import logo from "../../assets/images/logo.svg"
 import { completeSetup } from "../../context/commActions"
+import { useOnMountHistory } from "../../context/hooks/useOnMount"
 
 const SetupDonePage = () => {
+    const history: any = useOnMountHistory()
     const [confettiActive, setConfettiActive] = useState(false)
+
     useEffect(() => {
         setConfettiActive(true)
-        completeSetup()
-    }, [])
+
+        let sendNotification = true
+        if (
+            !history.location ||
+            !history.location.state ||
+            !history.location.state.doSendNotification
+        ) {
+            sendNotification = false
+        }
+        completeSetup(!sendNotification)
```

Comment:
Why is this `!` ?? I mean why  `completeSetup(!sendNotification)` and not  `completeSetup(sendNotification)`